functions.php

```diff
@@ -449,20 +449,30 @@ function interconnection_change_publish_button_js() {
  * so we can replace translated content in the loop. This allows
  * us to display English posts if the posts are not translated.
  *
- * This function also removes the sticky post from the homepage loop.
- *
  * @param WP_Query $query The WP_Query instance (passed by reference).
  */
 function interconnection_modify_polylang_query( $query ) {
 	// Query English posts only.
 	if ( function_exists( 'pll__' ) && ! is_admin() && ! is_singular() && $query->is_main_query() ) {
 		$languages = pll_default_language() . ',' . pll_current_language();
 		$query->set( 'lang', $languages );
-
-		// Remove sticky posts from homepage loop.
-		if ( is_home() ) {
-			$query->set( 'post__not_in', get_option( 'sticky_posts' ) ); // phpcs:ignore
-		}
 	}
 }
 //add_action( 'pre_get_posts', 'interconnection_modify_polylang_query' );
```

Comment:
Suddenly, it all makes sense

Reply:
Yes, I realized this after our conversation!